requirements.txt

```diff
@@ -28,11 +28,6 @@ requests==2.28.2
 sentry-sdk==1.16.0
 whitenoise==6.4.0
 
-# issue #3243
```

Comment:
Ah, I know this one - maybe a rebase on main will make some of these other commits go away.